examples/sparse/hypergraphatt.py

```diff
@@ -112,5 +123,13 @@ def main():
                 refresh=False,
             )
 
-if __name__ == '__main__':
-    main()
+    print(f"Test acc: {test_acc:.3f}")
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Hypergraph Attention Example")
```

Comment:
Why do we still use `argparse` only for this example?

Reply:
Because it's too slow for unit test. Training 500 epochs takes 10+ mins on CPU.